### PR TITLE
Update CitizenHut Recipe

### PR DESCRIPTION
**Pull request by Kostronor**
Review please @Minecolonies/maintainers

### CODE REVIEW

**Comment by Kostronor** [comment]
Sure, just edit it in, I have no IDE open at the moment

**Comment by Raycoms** [comment]
:shipit: now

**Comment by Kostronor** [comment]
@cltnschlosser

File: src/main/java/com/minecolonies/util/RecipeHandler.java [hunk 1]
```diff
@@ -61,6 +61,11 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
         if (supplyChests)
         {
             GameRegistry.addRecipe(new ItemStack(ModItems.supplyChest, 1), "B B", "BBB", 'B', Items.BOAT);
```
Comment (cltnschlosser) [inline src/main/java/com/minecolonies/util/RecipeHandler.java]: Can we use the ore dictionary for this?

File: src/main/java/com/minecolonies/entity/ai/minimal/EntityAIOpenFenceGate.java [hunk 2]
```diff
@@ -77,7 +80,8 @@ private void toggleDoor(boolean open)
         if (iblockstate.getBlock() == this.gateBlock && (iblockstate.getValue(BlockFenceGate.OPEN)) != open)
         {
             this.theEntity.worldObj.setBlockState(this.gatePosition, iblockstate.withProperty(BlockFenceGate.OPEN, open), 2);
-            this.theEntity.worldObj.playEvent((EntityPlayer) null, open ? OPEN_SOUND : CLOSE_SOUND, this.gatePosition, 0);
+            SoundEvent openCloseSound = open ? SoundEvents.BLOCK_FENCE_GATE_OPEN : SoundEvents.BLOCK_FENCE_GATE_CLOSE;
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/entity/ai/minimal/EntityAIOpenFenceGate.java]: MINOR Local variable 'openCloseSound' could be declared final [rule](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)

File: src/main/java/com/minecolonies/entity/ai/minimal/EntityAIOpenFenceGate.java [hunk 3]
```diff
@@ -1,9 +1,12 @@
 package com.minecolonies.entity.ai.minimal;
 
+import com.minecolonies.util.SoundUtils;
 import net.minecraft.block.BlockFenceGate;
 import net.minecraft.block.state.IBlockState;
 import net.minecraft.entity.EntityLiving;
 import net.minecraft.entity.player.EntityPlayer;
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/entity/ai/minimal/EntityAIOpenFenceGate.java]: MINOR Remove this unused import 'net.minecraft.entity.player.EntityPlayer'. [rule](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 4 issues
* MAJOR 2 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [EntityAIOpenFenceGate.java#L24](https://github.com/Minecolonies/minecolonies/blob/03d4574d745aedb6a7ee1707458aa748dfabf05e/src/main/java/com/minecolonies/entity/ai/minimal/EntityAIOpenFenceGate.java#L24): Remove this unused "OPEN_SOUND" private field. [rule](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)
1. MAJOR [EntityAIOpenFenceGate.java#L28](https://github.com/Minecolonies/minecolonies/blob/03d4574d745aedb6a7ee1707458aa748dfabf05e/src/main/java/com/minecolonies/entity/ai/minimal/EntityAIOpenFenceGate.java#L28): Remove this unused "CLOSE_SOUND" private field. [rule](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)

**Pull request merged**